### ISSUE TITLE
Error in HTML export

### ISSUE THREAD

**Issue by vhk28457** [giuspen/cherrytree#112]
Version 0.36.5 on Windows 7.
When I'm trying to export my notes to HTML, I'm getting an error:
```
Traceback (most recent call last):
  File "core.pyc", line 2199, in export_to_html
  File "exports.pyc", line 549, in nodes_all_export_to_html
  File "exports.pyc", line 557, in nodes_all_export_to_html_iter
  File "exports.pyc", line 621, in node_export_to_html
  File "exports.pyc", line 814, in html_get_from_treestore_node
  File "exports.pyc", line 853, in html_process_slot
  File "exports.pyc", line 895, in html_text_serialize
  File "exports.pyc", line 53, in rgb_48_to_48_no_white
ValueError: invalid literal for int() with base 16: ''
```

It took me more than two hours to figure out, what exactly causes this error. Here is base64-encoded ZIP with my settings and test note. Sorry for this format, GitHub says "Unfortunately, we don’t support that file type".
```
UEsDBBQAAAAAAK90VUgAAAAAAAAAAAAAAAALAAAAY2hlcnJ5dHJlZS9QSwMEFAAAAAgAwnRVSDpd
Nt7VBQAAqAsAABUAAABjaGVycnl0cmVlL2NvbmZpZy5jZmdUUkFu3DAMvOsVAXoOsNk0QC+99gFF
ewoCgpK4MbGyJIjceJ3XR1Rgb6KLzRFHMxzpWRSVXtyJE0Hkdvf77vM/40xWaCnJY4M3FvbJoH/t
Qm7hDCww45Vnfqf4Fa9FWLlkuHb0/td3bP3EpoqZosG9Pj4dXC6RoKJOvT64cGlS2s4aWKNAWSGW
YBxXOZw3x4nzGXStZm8hL06msoAdOeaAhJ5S3/uDSWgfiUP3I929WXCn1/75cfqynP8GHfpy7lkb
9bwaiQJdK4SS0uCbmoAv5Tzbxq428KFltoMzPjR+nRSEI+19YaLWVlgaVlg4jiAeHk2RImtpL07W
rHiFqXOT8Tmbv5qQ873SVR1etEDvGi3mKhbxqGFyomsikK4xbjUkFOHgpFJK0NEeZcJxXLvA3/+O
Mnoj3PZ3nyPaxJkgX2ZPTbarl4rBJs2ihBEUvewkK/apfjrMYSpty/7h4Gj29uo25PHoTGGEUfuY
m8RWd5FIWe0pHZ/G2DdkNDaF4XOZWAmGs5uZD0rKZidhIIrC+z6FG9xB2LrzNYwhTYEiTahtBlSW
DcEWdvwFWn8IQSFBjOFHggJuhv3wDudJ7DRlEk1ItJtO7rmzuOeeb8zDYnAzcE/2I0j4nCLy5kEl
K5OrAI3ovz++gpSmcJOVwEpYD7AeYXVh9eHW4NbhNmANYU1QeqUj+oZyn05gN2HXYTdg12C3YC/g
NOHU4TTg1OC06JqO0d6gPUN7hU4Z1WdUB6j26JR+0hmd0y+6we39trNdotLdfqBSR+UOlTFrsXfW
21nsiQ3ZaFdlUzZjc7ZgS7ZmG7gruGu4G3gDeBa8ErwXeGMpr+ZuDJIWg8Sip1JB031OFN2UMwbR
Fb6eSDoW0WORs4DZvEzUnFLQroWVPr1c8D3OFwxdVlIc//1yMhpRM0bxKJI/Og5eqcOtxfRF1DDV
y7DXyKVV8tdu6TzlM5s0igkpPIjonsRjcVHMqpxCHuh9VbTKplZUcyJVe1FwE44r6jqnVE4S5bcg
aP5BupCD6BOVYxPe5C8UZzchBT+ZfJdZLb1twzD4nl9RILdBB+fZNLc8GqRA0xaJt1thyBadCpUl
T5LTbL9+pGzHLmYdTPIjJVo0H+bLhRxr+MyoqtAkmnyTJIUJxYgL0cmTQupg3LOmmk/WoRjmRnuH
R6FPCdEIHIw2IafCmRagBX6BFVzzuzUGgzD6hhbbmMpKsHcv8EUQRUAZ694pHQXkSV2co/C0sn51
HvhvemmURRnJOj2IaA1KrsB7wEA435pH0bLZetlstxze57SIr+U8Gkd5o0c4F2IBc+SRq+34jN4w
h3nOEZ+N+Rjfi3T+MJouhyN4QE3SQ8/S5fAhApSgnONCvamgtRymOS3cZ0ZrQPcdflvqau0VTPEh
LzuUekxA70fZYrIAencoZQqhi5S8IX9G0w6l3CD0Pu8WBqCQLnu/tcpKJhfIUO0HO/ELMAclt9wb
y35qYdgRhOnJYguwEuIFvWrpU5Ui2zc8YQneSwGkwAhzj9dypVRNb4xSxHT6O6nFkyawR7pA34Sr
M5c6UGuefYYjcJMDJRe4reTKnHs7kllszmcFx9fAPArpe/iea6HgqeBnaOgYk6ClN2iwNlf2WKQ7
iWragfUN9owX25Cr0OpYDcevG7Y3Vv412nN1rFT/QtaVUuBBPEvn2UvorQ0Tm60hoqd8hMJcYBeq
Kg0CbEMJQ199tqbSouYRh4avP3TDXf9ISkX2hK7IjAIZSiSwk7fyE/wHGp4/2H7E9mO2n7BTwTEg
p6oE6zIrS4902lBt0vc3x3Aa68f7+PDc0m/bXUvGV3+olJelgla0ibf9eON14O4Amr1ZyMGCzsDh
EOS85X+6ccRz6xOjkw4IZbBykIRhQdxUqVg6foH/BMmFKxTOBvUkdAHres1hYEEZLmjwTBR3txmj
MCKhBpe43uCRYgQqmgrL1sVW1pT9SXcqOv27kv6mp2QhfYLBMvXw5aFsp+PBP1BLAwQKAAAAAABI
qktIOIy/fAIAAAACAAAADwAAAGNoZXJyeXRyZWUvbGFuZ3J1UEsDBBQAAAAIAENzVUjkVQp7mgAA
ANIAAAAIAAAAdGVzdC5jdGRNjjsOwjAQRK+yWiQ6fqED2+m4RmSczUc4XrK2UXJ7QoqI14w0o5Ge
KqfBw4ck9hw0Xo5nhNIo15HInITIqMA1QbADaUwUE8JbuK28Da1Gl2Pi4eDYs0QEIVtz8LPGh/WR
EJJto0aEHPoxU9XXGq8FGiW966pEU4Knda9WOIdl2jUrCA0LbWWxgmY/Zk73mzptZ6NOP7kl/ny/
UEsBAj8AFAAAAAAAr3RVSAAAAAAAAAAAAAAAAAsAJAAAAAAAAAAwAAAAAAAAAGNoZXJyeXRyZWUv
CgAgAAAAAAABABgAALkcP5xs0QHb45MbnWzRATNciRudbNEBUEsBAj8AFAAAAAgAwnRVSDpdNt7V
BQAAqAsAABUAJAAAAAAAAAAgAAAAKQAAAGNoZXJyeXRyZWUvY29uZmlnLmNmZwoAIAAAAAAAAQAY
AAC2YFOcbNEBvG+JG51s0QG8b4kbnWzRAVBLAQI/AAoAAAAAAEiqS0g4jL98AgAAAAIAAAAPACQA
AAAAAAAAIAAAADEGAABjaGVycnl0cmVlL2xhbmcKACAAAAAAAAEAGAAApIOT+GTRAdvjkxudbNEB
2+OTG51s0QFQSwECPwAUAAAACABDc1VI5FUKe5oAAADSAAAACAAkAAAAAAAAACAAAABgBgAAdGVz
dC5jdGQKACAAAAAAAAEAGAAAm2qnmmzRAWP3kxudbNEBY/eTG51s0QFQSwUGAAAAAAQABAB/AQAA
IAcAAAAA
```

**Comment by giuspen** [comment]
Please send me by mail your document tree or a subset/example enough for me to reproduce the issue. You can find my mail in cherrytree help--about dialog

**Comment by vhk28457** [comment]
I've sent a mail to you.

**Comment by giuspen** [comment]
Thank you I should have fixed the bug. Can you try the latest git version or you need me to create a package for you?

**Comment by vhk28457** [comment]
@giuspen, it fixed in 9fb8c35, thanks!
